app/lib/task/backend.dart

```diff
@@ -454,6 +455,20 @@ class TaskBackend {
         return false;
       }
 
+      state.abortedTokens = [
+        ...state.versions!.entries
+            .where((e) => deselectedVersions.contains(e.key))
+            .map((e) => e.value)
+            .where((vs) => vs.secretToken != null)
+            .map(
+              (vs) => AbortedTokenInfo(
+                token: vs.secretToken!,
+                expires: vs.scheduled.add(maxTaskExecutionTime),
```

Comment:
We should have stored `scheduled` instead of `expires`, not that it matters more, but it would have been 10% more canonical.

Anyways, I don't think it matters :D

Reply:
We could replace it, but I'm not sure it is worth it...